R/executeDqChecks.R

```diff
@@ -77,11 +78,11 @@ executeDqChecks <- function(connectionDetails,
                             fieldCheckThresholdLoc = "default",
                             conceptCheckThresholdLoc = "default") {
   # Check input -------------------------------------------------------------------------------------------------------------------
-  if (!("connectionDetails" %in% class(connectionDetails))) {
-    stop("connectionDetails must be an object of class 'connectionDetails'.")
+  if (!any(class(connectionDetails) %in% c("connectionDetails", "ConnectionDetails"))) {
+    stop("connectionDetails must be an object of class 'connectionDetails' or 'ConnectionDetails'.")
```

Comment:
@schuemie the recent DatabaseConnector upgrade was causing `executeDqChecks` to fail at this input check, as it seems the `connectionDetails` created by the new DatabaseConnector version is of `ConnectionDetails` class, not `connectionDetails`?  

my change here was inspired by your Eunomia PR [here](https://github.com/OHDSI/Eunomia/pull/36).  is this OK?  or should we require DatabaseConnector >6.0.0?  or something else?  i'm not familiar with what the change in capitalization represents so want to make sure i do the right thing here - thanks.

Reply:
The change you have is fine. Sorry about that. I had no idea that simple renaming would have such wide impact.